### DISCUSSION TITLE
Как узнать индексируется ли SCU по запросу?

### DISCUSSION THREAD

**Discussion by Zeazurus** [glmn/wb-private-api#42] [Предложения]
С помощью WBPrivateAPI.search() и WBCatalog.getPosition() можно получить текущую позицию SCU по запросу. Но не всегда артикул находится в диапазоне, который может быть получен. В конечном счете мы получаем -1, даже если нас по этому запросу можно найти. Это понятно, функция выполняет то, что должна, не больше. 

А можно как-нибудь получить WBCatalog, но отфильтрованный по бренду или продавцу, чтобы уже в нем можно было посмотреть, есть ли мы по запросу или нет? Я смотрел в сторону WBPrivateAPI.searchCustomFilters, но он возвращает только общее число товаров и конкретный SCU невозможно отследить.

**Comment by glmn** [comment]
Вопрос предельно понятен. Функционала такого нет.
Тема данная мне интересна, готовлю PR 🚀

**Reply by glmn** [reply]
Добавил PR #43
Функционал будет добавлен в версии 0.5.9

**Reply by glmn** [reply]
Версия `0.5.9` уже доступна в `npm`

**Reply by glmn** [reply]
Спасибо за предложение @Zeazurus 🎉 🚀

**Reply by Zeazurus** [reply]
Спасибо за реализацию @glmn =)
